src/components/Navbar.jsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure code quality.

Reply:
I usually don't disable linters as you mentioned but that has to do with Material UI  syntax 
![image](https://user-images.githubusercontent.com/87689547/147312014-0e0dd833-e0b7-441c-9e90-3197326aa8a9.png)
 as you can see here there is no way I could spread it

Reply:
- I was checking the props that you were passing `params` and it brings:

![image](https://user-images.githubusercontent.com/79876959/147319981-5b58657f-4005-4cf7-9c52-fd7b59c8b363.png)

So, a solution could be passing the props like:

![image](https://user-images.githubusercontent.com/79876959/147320022-47989332-b325-4fbe-9b69-a03c2b376bc5.png)

However, it would add a new linter issue because of the name of props. Only for this situation, the spread operator seems a better option than passing each prop. 👍

Reply:
Thanks for investigating, really appreciate your effort and your feedback